src/Middleware/HttpLogging/src/HttpLoggingFields.cs

```diff
@@ -175,5 +175,8 @@ public enum HttpLoggingFields : long
     /// the entire request and response body up to the <see cref="HttpLoggingOptions.RequestBodyLogLimit"/>
     /// and <see cref="HttpLoggingOptions.ResponseBodyLogLimit"/>.
     /// </summary>
+    /// <remarks>
+    /// This will not include <see cref="RequestQuery"/> as it may contain private information. 
```

Comment:
How about something like:

```suggestion
    /// The HTTP Request <see cref="HttpRequest.QueryString"/> is not included with this flag as it may contain private information.
    /// If desired, it should be explicitly specified with <see cref="RequestQuery"/>.
```

Reply:
I think having the remark is sufficient, adding the extra flag seems clunky.